pkg/api/dtos/frontend_settings.go

```diff
@@ -230,6 +230,9 @@ type FrontendSettingsDTO struct {
 
 	LoginError string `json:"loginError,omitempty"`
 
+	// The namespace to use for this user
+	K8sNamespace string `json:"namespace,omitempty"`
```

Comment:
not a big deal, but should this be `Namespace` to keep things consistent?
```suggestion
	// The K8s namespace to use for this user
	Namespace string `json:"namespace,omitempty"`
```